app/Services/Plans/PlanService.php

```diff
@@ -455,6 +455,8 @@ private function translatePlaylist(
         $total_translated_items = 0;
         
         if (isset($playlist->items)) {
+            $books_targe_bible = $bible->books->keyBy('book_id');
```

Comment:
change to books_target_bible throughout

Reply:
@bradflood , I have fixed it. see: https://github.com/faithcomesbyhearing/dbp/pull/934/files#diff-d5564477e2a1134b703453bf7b1614170d3265d72f2f101a11e809e44496ee25R478